### ISSUE TITLE
training set question

### ISSUE THREAD

**Issue by ljljlj02** [davidiommi/Pytorch-Unet3D-single_channel#1]
Hi, I am not  understand about the data set setting of your program in the process of running. May I ask what data set this program applies to?Why do data_1 include A images and B labels in your training set?Shouldn't it be A images and A labels?
  Also, Can it run under Win10?
  Any help will be greatly appreciated

**Comment by davidiommi** [comment]
It's a general script for segmentation. 
- I used it for carotid artery segmentation in this case. In this repository https://github.com/davidiommi/3D_Prostate_segmentation_pytorch I used it with the Promise2012 prostate dataset ( which is public). 
- With data_1, data_2  I meant the patients list and A= image, B= label for each patient case. In the prostate repository you can read it explicitly. ( I modified the Read.me file to make it clear now). 
- I did run the inference on my Win 10 with Pycharm, but the training on Linux. I guess it won't be a problem doing both with Win 10.

**Comment by ljljlj02** [comment]
Interesting. I'll try it. Thank you!

**Comment by chenjingyi-star** [comment]
Why is it that when I train my model on my computer it automatically reboots after two runs?I would like to ask you if there is a similar situation.I only have a 2080ti GPU and a Ubuntu18.04 system.Is the configuration insufficient?
